dlog/commitment/src/commitment.rs

```diff
@@ -80,7 +83,7 @@ impl<C: AffineCurve> PolyComm<C> {
                 if com.len() == 0 || elm.len() == 0 {
                     Vec::new()
                 } else {
-                    let n = com.iter().map(|c| c.unshifted.len()).max().unwrap();
+                    let n = Iterator::max(com.iter().map(|c| c.unshifted.len())).unwrap();
```

Comment:
here is the change. The file was not properly formatted hence the other no-op changes